cmd/ll2go/main.go

```diff
@@ -218,8 +218,8 @@ func ll2go(llPath string, funcNames map[string]bool) (*ast.File, error) {
 			underlying = "int32"
 		case intSize < 64:
 			underlying = "int64"
-		default:
-			return nil, errors.Errorf("support for integer type with bit size %d not yet implemented", intSize)
+		default: // math.Big
+			underlying =  "Int"
```

Comment:
Have you tried to use this in practise? I would reckon a lot of the could would have to be updated since [math/big/Int](https://golang.org/pkg/math/big/#Int) does not use `+` and `-` binary operators for addition and subtraction, but rather `x.Add` and `x.Sub` method calls. Also, the underlying type would be `*big.Int` I would assume, as that is the [qualified identifier](https://golang.org/ref/spec#Qualified_identifiers) for the type.

I would personally prefer that we keep this panic until proper support for `big.Int` is implemented. Preferably in a separate PR as the implementation may be tricky and require more extensive review and testing.